build-tools/packages/build-cli/src/commands/bump.ts

```diff
@@ -78,6 +78,11 @@ export default class BumpCommand extends BaseCommand<typeof BumpCommand> {
 				'Controls the type of dependency that is used between packages within the release group. Use "" (the empty string) to indicate exact dependencies. Use the workspace:-prefixed values to set interdependencies using the workspace protocol. The interdependency range will be set to the workspace string specified.',
 			options: [...RangeOperators, ...WorkspaceRanges],
 		}),
+		onlyUpdateWorkspaceDeps: Flags.boolean({
+			description:
```

Comment:
```suggestion
			allowNo: true,
			description:
```

Reply:
This enables the `--no-NAME` flag to disable the flag.

Reply:
You should also add an env argument so it'll read from the env var if set. maybe FLUB_BUMP_ONLY_UPDATE_WORKSPACE_DEPS.

Reply:
Hmmm, probably better to invert the name and logic, you just pass the flag to enable the old behavior instead of passing a disable flag to enable the old behavior, The flag would be something like updateAllDeps.

Reply:
Ah, forgot the env var bit, I'll update. I think the current implementation already only needs the flag for the old behavior, since it defaults to true, but I like the suggestion, I'll change that too.

Reply:
Confirmed that `--updateAllDeps` or `FLUB_BUMP_UPDATE_ALL_DEPS=1 <command>` both use the current logic.